### DISCUSSION TITLE
No updates found for some NuGet packages from Artifactory feed

### DISCUSSION THREAD

**Discussion by ap0llo** [renovatebot/renovate#28089] [Request Help]
### What would you like help with?

Other

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Platform GitLab, Docker image: renovate/renovate:37.31

### Please tell us more about your question or problem

Disclaimer: I'm not sure if this is a problem with renovate or in the configuration I'm using, but I would be grateful for some pointers on how to track down the issue

- I have a repository hosted on a GitLab server that consumes packages from a NuGet feed hosted on an internal Artifactory server.
- Renovate mostly works as expected and creates update MRs if a new package versions if uploaded to Artifactory
- However, for two specific packages, no updates are created, even though the dependency is correctly detected (it shows up under *Detected dependencies* in the dependency dashboard). 

I already worked through the Debug logs to find some issue, but could not find anything relevant.

There is an error when it tries to look up the package ion nuget.org (that's expected) but that error is also there for packages for which updating works, other than that I could not find any errors, it just seems like there aren't just any updates for that package,

Is there some additional logging option I could enable (e.g. to see the exact request renovate sends to the Artifactory server)?
Do you have any hints what I should look into?

Any tips you have would be greatly appreciated.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Have a look in "http statistics" debug log message to see which URLs are queried for one of those dependencies and then try querying them manually with e.g. `curl` to see what data is returned. Then compare to one that works
